src/main/java/org/example/MCQ's Test.java

```diff
@@ -0,0 +1,742 @@
+import java.awt.Color;
```

Comment:
- Place this file inside _org.examples.awt_ package
- File name should be same as name of public class and in CamelCase